.github/workflows/publish-go-tester-task.yml

```diff
@@ -119,6 +122,9 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           version: 3.x
 
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v3
```

Comment:
Please review the alternative approach I proposed at https://github.com/arduino/serial-discovery/pull/94

I have verified that the same solution works in this project.